src/adapters/Cornerstone3D/MeasurementReport.js

```diff
@@ -100,31 +172,43 @@ export default class MeasurementReport {
             ReferencedFrameNumber
         } = ReferencedSOPSequence;
 
+        const referencedImageId =
+            sopInstanceUIDToImageIdMap[ReferencedSOPInstanceUID];
+        const imagePlaneModule = metadata.get(
+            "imagePlaneModule",
+            referencedImageId
+        );
+
+        const finding = findingGroup
+            ? findingGroup.ConceptCodeSequence[0]
+            : undefined;
+        const findingSites = findingSiteGroups.map(fsg => {
+            return { ...fsg.ConceptCodeSequence[0] };
+        });
+
         const defaultState = {
-            sopInstanceUid: ReferencedSOPInstanceUID,
-            frameIndex: ReferencedFrameNumber || 1,
-            complete: true,
-            finding: findingGroup
-                ? findingGroup.ConceptCodeSequence
-                : undefined,
-            findingSites: findingSiteGroups.map(fsg => {
-                return { ...fsg.ConceptCodeSequence };
-            })
+            annotation: {
+                annotationUID: DicomMetaDictionary.uid(),
```

Comment:
shouldn't this be the same as annotationUID in cs3d? (I don't konw, just asking)

Reply:
The annotationUIDs in CS3D are for internal referencing, this just generates one from the SR's data. We aren't currently encoding the annotationUID on save right now, its something we could do in the future. Maybe its useful, but i'm not sure. Happy to discuss at some point.